pkg/store/bucket.go

```diff
@@ -1260,11 +1260,12 @@ func newBucketIndexReader(ctx context.Context, logger log.Logger, block *bucketB
 }
 
 func (r *bucketIndexReader) lookupSymbol(o uint32) (string, error) {
-	s, ok := r.block.symbols[o]
-	if !ok {
-		return "", errors.Errorf("bucketIndexReader: unknown symbol offset %d", o)
+	idx := int(o)
+	if idx < len(r.block.symbols) {
```

Comment:
Consider indenting errow flow as per https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

i.e. checking for  `if idx  >= len(..)` and returning and error